### PR TITLE
Deployment script and instructions

### PR DESCRIPTION
**Pull request by EdNoepel**
**Changes**
- Parameterized AJNA token address so we can deploy to multiple chains.
- Created a shell script to manage factory contract deployment.
- Updated README with deployment instructions.

### CODE REVIEW

File: tests/forge/interactions/PurchaseQuoteWithExternalLiquidity.t.sol [hunk 1]
```diff
@@ -12,6 +12,7 @@ import "./BalancerUniswapExample.sol";
 contract PurchaseQuoteWithExternalLiquidityTest is Test {
     address constant WETH     = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
     address constant USDC     = 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48;
+    address constant AJNA     = 0x9a96ec9B57Fb64FbC60B423d1f4da7691Bd35079;
```
Comment (MikeHathaway) [inline tests/forge/interactions/PurchaseQuoteWithExternalLiquidity.t.sol:15]: Any reason we couldn't use `_ajna` from DSTestPlus?

**Review by MikeHathaway — APPROVED** [review]
LGTM

**Review by grandizzy — APPROVED** [review]
Lgtm, great stuff!

File: deploy.sh [hunk 2]
```diff
@@ -27,18 +27,35 @@ do
...
-factories=( ERC20PoolFactory ERC721PoolFactory )
+factories=( 'erc20,ERC20PoolFactory' 'erc721,ERC721PoolFactory' )
 for contract in "${factories[@]}"
 do
+    IFS=',' read -r -a arr <<< ${contract}
+    dir=${arr[0]}
+    name=${arr[1]}
     createfactory="forge create --rpc-url ${ETH_RPC_URL:?} --keystore ${DEPLOY_KEY:?} --password ${password:?} \
-        src/erc20/ERC20PoolFactory.sol:ERC20PoolFactory --constructor-args ${AJNA_TOKEN:?} ${linkage}"
+        src/$dir/$name.sol:$name --constructor-args ${AJNA_TOKEN:?} ${linkage}"
```
Comment (grandizzy) [inline deploy.sh:37]: will keep in mind that script needs to be updated when restructure code and move pool and factory contracts in root dir of src repo

**Review by grandizzy — APPROVED** [review]
👍

**Pull request merged**